src/Profile.php

```diff
@@ -66,6 +66,8 @@ public static function fromUuid($uuid, ApiClient $apiClient = null)
             }
         }
 
+        $profile->history = $apiClient->uuidHistory($uuid);
```

Comment:
I'm mostly for this.  Some worries based on whether or not the user accessing the variable knows that they'll be making a web call though..